lib/customs.js

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var P = require('./promise')
+var Pool = require('./pool')
+
+function Customs() {
+  this.pool = new Pool("http://127.0.0.1:7000", { timeout: 1000 })
+}
+
+Customs.prototype.checkAuthenticated = function (ip, token, uid) {
+  return this.pool.post(
+      '/checkAuthenticated',
+      {
+        ip: ip,
+        token: token,
+        uid: uid
+      }
+  ) 
```

Comment:
@larissagaulia I guess this also needs to use the `tooManyRequests` error: https://github.com/mozilla/fxa-auth-server/blob/7ca2ffba99e09a1e0025baa26bbaca35598d73bd/lib/customs.js#L51